serverless/cmd/client/client.go

```diff
@@ -64,11 +85,13 @@ func main() {
 		glog.Exitf("Invalid log URL: %q", err)
 	}
 
-	// TODO(al) derive this from log pub key
-	const logID = "test"
+	// Derive logID from log public key
+	v, _ := note.NewVerifier(pubKey)
```

Comment:
Should be checking `err` here I suspect.

Reply:
done